### PR TITLE
feat(@desktop/wallet): New Account header

### LINKED ISSUE #10330: New Account header
As per design :

1. New header - [https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=12970-505648&t=zqIHTe3GG0EC5YOp-4](https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=12970-505648&t=zqIHTe3GG0EC5YOp-4)
2. Clicking on address on header launches receive modal [https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=12934-425367&t=zqIHTe3GG0EC5YOp-4](https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=12934-425367&t=zqIHTe3GG0EC5YOp-4)
3. New network filter https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=13179-346563&t=Ujxp0sa5eIXJchss-4

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #10330

### What does the PR do

Implements:
1. New header - https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=12970-505648&t=zqIHTe3GG0EC5YOp-4
2. Clicking on address on header launches receive modal https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=12934-425367&t=zqIHTe3GG0EC5YOp-4
3. New network filter https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=13179-346563&t=Ujxp0sa5eIXJchss-4

### Affected areas

Wallet Accounts View

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://user-images.githubusercontent.com/60327365/237047822-0d7e6914-1468-408f-99e4-c4073851c897.mov

### CODE REVIEW

File: ui/app/AppLayouts/Wallet/stores/RootStore.qml [hunk 1]
```diff
@@ -197,4 +197,13 @@ QtObject {
...
         walletSection.runEditAccountPopup(address)
     }
+
+    function getAllNetworksSupportedString() {
+        let result = ""
+        for(var i = 0; i < allNetworks.count; i++) {
+            let shortName = allNetworks.rowData(i, "shortName")
+            result += shortName + ':'
+        }
+        return result
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/stores/RootStore.qml:207]: You could do this in NIM and export it straight from there :)

Accessing `rowData` from here sounds like a hack... Besides in NIM is gonna be faster

Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Wallet/stores/RootStore.qml]: done @caybro :)

File: ui/app/AppLayouts/Wallet/controls/NetworkFilter.qml [hunk 2]
```diff
@@ -51,103 +46,85 @@ Item {
...
+            color: Theme.palette.baseColor1
+            visible: !!text
+        }
+        Row {
+            spacing: -4
+            visible: !d.allSelected && chainRepeater.count > 0
+            Repeater {
+                id: chainRepeater
+                model: root.enabledNetworks
+                delegate: Image {
```
Comment (Cuteivist) [inline ui/app/AppLayouts/Wallet/controls/NetworkFilter.qml:94]: Figma has border around second and any next item
image

In app there is no border
image

Other than that really awesome job!

**Review by caybro — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Wallet/controls/NetworkFilter.qml (hunk 2 above)
Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Wallet/controls/NetworkFilter.qml:94]: image
Hey thanks for pointing it out @Cuteivist. Fixed as you can see in screenshot above :)

**Comment by Cuteivist** [comment]
LGTM. Great job!

**Review by Cuteivist — APPROVED** [review]

**Pull request merged**